packages/qwik/src/core/shared/shared-serialization.ts

```diff
@@ -577,9 +577,11 @@ type SsrNode = {
 };
 
 type DomRef = {
-  id: string;
+  ssrNode: SsrNode;
 };
 
+let isDomRef: ((obj: unknown) => obj is DomRef) | null = null;
+
```

Comment:
I am not sure if this is a good idea

Reply:
I don't see why not, although you could initially put the false function in there